### ISSUE TITLE
Scientific and numeral libraries

### ISSUE THREAD

**Issue by vindarel** [ciel-lang/CIEL#5]

**Comment by jgarte** [comment]
How about packages from [lisp-stat](https://lisp-stat.dev/)?

**Comment by vindarel** [comment]
Absolutely. Then maybe a ciel-stat super package. Or choose the indispensable libs to include in CIEL, not surcharging it too much.

**Comment by jgarte** [comment]
Looks like there was a book written about the library:

https://play.google.com/store/books/details?id=ie_KKAamFM4C&rdid=book-ie_KKAamFM4C&rdot=1&source=gbs_vpt_read&pcampaignid=books_booksearch_viewport&pli=1

**Comment by vindarel** [comment]
That's a confusing part about the lisp-stat: it takes good parts from L. Tierney's XLisp-Stat: https://lisp-stat.dev/docs/overview/

> XLisp-Stat code is unlikely to run except in trivial cases.

This book is about xlisp-stat. I read:

> At the time of writing, there is one implementation of lisp-stat, called XLISP-STAT […] based on the XLISP dialect of Lisp, a subset of Common Lisp.
